starboard/android/shared/media_codec_bridge_eradicator.h

```diff
@@ -40,14 +40,22 @@ class MediaCodecBridgeEradicator {
 
   bool Destroy(jobject j_media_codec_bridge,
                jobject j_reused_get_output_format_result);
-  void WaitForPendingDestructions();
+  bool WaitForPendingDestructions();
   bool IsEnabled() const { return is_enabled_.load(); }
   void SetEnabled(bool enabled) { is_enabled_.store(enabled); }
+  bool SetTimeoutSeconds(int timeout_seconds) {
+    timeout_seconds_ = timeout_seconds;
+  }
+  bool GetTimeoutSeconds() { return timeout_seconds_; }
 
  private:
   static void* DestroyMediaCodecBridge(void* context);
 
   atomic_bool is_enabled_{false};  // false by default
```

Comment:
nit: the comment is no longer necessary.

Reply:
removed